app/assets/javascripts/common/toggleScrollToPosition.js

```diff
@@ -0,0 +1,31 @@
+// Check for scroll position and scroll
+window.addEventListener("DOMContentLoaded", (event) => {
+  if (doesYPositionExistInLocalStorage()) {
+    scrollToYPosition();
+    removeYPositionLocalStorage();
+  }
+});
```

Comment:
There is a check to see if a scroll position exists and if so scroll to that position and remove the position from localStorage